2021-03-09 23:57:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-09 23:57:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-09 23:57:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-09 23:57:34 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:29 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:34 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:39 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:44 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:49 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:54 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:02:59 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:03:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:03:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:03:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:15 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:20 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:30 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:35 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:40 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:45 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:05:55 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:06:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:06:05 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:06:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:06:15 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:06:20 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:06:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:06:30 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:07:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:07:45 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:07:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:14:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:14:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:14:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:14:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:14:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:14:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:15:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:16:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:17:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:18:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:19:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:20:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:21:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:22:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:23:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:24:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:25:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:26:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:27:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:28:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:29:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:30:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:31:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:32:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:33:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:34:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:35:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:36:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:37:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:38:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:39:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:39:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:39:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:43:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:43:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:43:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:43:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:43:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:44:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:44:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:45:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:45:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:45:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:45:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:45:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:45:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:46:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:47:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:47:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:47:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:47:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:47:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:48:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:48:05 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:54 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:49:59 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:14 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:19 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:29 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:34 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:40 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:45 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:50:55 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:05 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:15 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:20 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:29 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:45 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:55 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:51:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:05 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:09 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:29 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:29 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:52:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:30 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:53:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:30 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:54:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:55:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:56:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:57:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:44 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:55 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:58:59 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 00:59:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:00:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:05 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:15 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:34 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:01:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:35 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:40 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:45 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:55 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:02:59 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:04 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:08 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:15 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:20 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:30 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:35 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:40 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:45 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:03:55 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:05 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:15 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:20 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:30 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:35 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:40 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:45 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:04:55 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:05 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:15 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:28 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:33 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:38 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:05:58 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:03 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:06:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:07:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:44 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:50 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:08:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:09:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:09:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:09:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:09:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:09:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:43 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:49 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:14:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:15:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:16:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:17:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:00 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:46 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:51 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:18:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:01 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:19:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:29:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:06 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:30:26 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:32:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:32:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:32:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:33:21 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:33:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:33:30 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:34:10 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:36:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:45:44 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:45:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:45:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:45:53 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:49:54 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:49:59 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:49:59 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:50:11 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:54:20 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:54:24 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:54:25 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:54:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:59:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 01:59:16 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:05:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:05:44 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:05:56 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:09:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:09:44 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:09:48 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:09:49 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:09:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:10:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:11:09 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:11:13 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:11:18 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:11:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:17:23 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:17:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:17:29 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:29:12 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:29:16 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:29:18 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:29:23 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:29:48 ERROR PsqlService:99 - Колонки accountId не найдено в этом ResultSet’’е.
2021-03-10 02:29:48 ERROR PsqlService:99 - Колонки accountId не найдено в этом ResultSet’’е.
2021-03-10 02:32:40 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:32:45 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:32:45 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:32:49 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:34:27 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:37:16 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:37:20 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:37:21 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:37:27 ERROR PsqlService:99 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:43:51 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:43:55 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:44:00 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:44:05 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:44:10 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:44:15 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:44:20 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:44:25 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:44:26 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:45:00 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:49:34 ERROR PsqlService:100 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:16 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:21 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:26 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:31 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:36 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:41 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:46 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:51 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:53:56 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:01 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:11 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:16 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:21 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:26 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:31 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:36 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:41 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:46 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:51 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:54:56 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:01 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:11 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:16 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:21 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:26 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:31 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:36 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:41 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:46 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:51 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:55:56 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:01 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:11 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:16 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:21 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:26 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:31 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:36 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:41 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:46 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:51 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:56:56 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:01 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:11 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:16 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:18 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:19 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:19 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:19 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:19 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 02:57:23 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:00:42 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:00:47 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:00:47 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:04:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:05:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:06:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:07:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:08:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:47 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:52 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:09:57 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:02 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:07 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:12 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:17 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:22 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:27 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:37 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:42 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:10:58 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:02 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:03 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:03 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:07 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:12 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:17 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:22 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:27 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:32 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:37 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:42 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:47 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:52 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:11:57 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:01 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:04 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:05 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:05 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:05 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:05 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:06 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:07 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:07 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:07 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:12 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:17 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:22 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:27 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:32 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:37 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:12:42 ERROR PsqlService:97 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-10 03:15:00 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:15:02 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:15:04 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:15:07 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:15:09 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:15:12 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:15:14 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:15:17 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:18:52 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:18:53 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:18:55 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:18:57 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:18:57 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:00 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:02 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:02 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:05 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:07 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:07 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:10 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:12 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:12 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:15 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:17 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:17 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:20 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:22 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:22 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:25 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:27 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:27 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:30 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:32 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:32 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:35 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:35 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:37 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:19:42 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:00 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:02 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:05 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:07 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:10 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:12 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:15 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:17 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:20 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:22 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:25 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:27 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:30 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:32 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:35 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:37 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:40 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:42 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:45 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:46 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:47 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:51 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 03:24:52 ERROR PsqlService:98 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:09 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:14 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:19 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:24 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:29 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:34 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:39 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:44 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:49 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:54 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:42:59 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:43:04 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:43:09 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:43:14 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:43:19 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:43:20 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:43:24 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 13:43:29 ERROR PsqlService:97 - ОШИБКА: отношение "pls" не существует
  Позиция: 15
2021-03-10 18:58:29 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:31 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:33 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:34 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:35 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:36 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:40 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:41 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:45 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:46 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:47 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:51 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:58:56 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:01 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:05 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:06 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:07 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:07 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:07 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:08 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 18:59:08 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:09 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:11 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:14 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:16 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:19 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:21 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:26 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:31 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:36 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:41 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:46 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:51 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-10 19:02:56 ERROR PsqlService:96 - ОШИБКА: столбец "halls.idh" должен фигурировать в предложении GROUP BY или использоваться в агрегатной функции
  Позиция: 8
2021-03-11 03:19:31 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-11 03:19:32 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-11 03:19:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-11 03:19:36 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-11 03:19:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-03-11 03:19:41 ERROR PsqlService:96 - ОШИБКА: отношение "halls" не существует
  Позиция: 15
2021-04-04 15:05:39 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:31)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:19:18 ERROR PsqlService:77 - ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
org.postgresql.util.PSQLException: ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at ru.job4j.cinema.service.PsqlService.buy(PsqlService.java:73)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:36)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:20:33 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:35)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:20:33 ERROR PsqlService:77 - ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
org.postgresql.util.PSQLException: ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at ru.job4j.cinema.service.PsqlService.buy(PsqlService.java:73)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:36)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:22:33 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:35)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:22:33 ERROR PsqlService:77 - ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
org.postgresql.util.PSQLException: ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at ru.job4j.cinema.service.PsqlService.buy(PsqlService.java:73)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:36)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:26:26 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:35)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:26:26 ERROR PsqlService:77 - ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
org.postgresql.util.PSQLException: ОШИБКА: INSERT или UPDATE в таблице "halls" нарушает ограничение внешнего ключа "halls_account_id_fkey"
  Подробности: Ключ (account_id)=(0) отсутствует в таблице "accounts".
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:124)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.executeUpdate(DelegatingPreparedStatement.java:136)
	at ru.job4j.cinema.service.PsqlService.buy(PsqlService.java:73)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:36)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:27:59 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:40)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:30:00 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:40)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 15:38:27 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(3, 3)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(3, 3)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:36)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:20:05 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:37:51 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:39:21 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:44:18 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:44:36 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:44:43 ERROR PsqlService:139 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:137)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:47:21 ERROR PaymentServlet:45 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:136)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:53:43 ERROR PaymentServlet:45 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:136)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:55:07 ERROR PaymentServlet:46 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(2, 2)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:136)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
2021-04-04 17:56:26 ERROR PaymentServlet:46 - ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
org.postgresql.util.PSQLException: ОШИБКА: повторяющееся значение ключа нарушает ограничение уникальности "orders_rowx_columnx_key"
  Подробности: Ключ "(rowx, columnx)=(1, 1)" уже существует.
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2505)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2241)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:310)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:447)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:368)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:158)
	at org.postgresql.jdbc.PgPreparedStatement.execute(PgPreparedStatement.java:147)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at org.apache.commons.dbcp2.DelegatingPreparedStatement.execute(DelegatingPreparedStatement.java:94)
	at ru.job4j.cinema.service.PsqlService.addOrders(PsqlService.java:136)
	at ru.job4j.cinema.controller.PaymentServlet.doPost(PaymentServlet.java:38)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:652)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:733)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:227)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:189)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:162)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:97)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:542)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:143)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:687)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:78)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:346)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:374)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:65)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:887)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1684)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.base/java.lang.Thread.run(Thread.java:832)
